app/components/gh-distribution-action-select.js

```diff
@@ -0,0 +1,47 @@
+import Component from '@glimmer/component';
+import {action} from '@ember/object';
+import {tracked} from '@glimmer/tracking';
+
+export default class GhDistributionActionSelect extends Component {
+    @tracked
+    distributionValue;
+
+    constructor() {
+        super(...arguments);
+
+        this.distributionValue = this.args.post.get('emailOnly')
```

Comment:
FYI, `.get()` is not needed anywhere except for accessing proxy objects which at the moment are just the `settings` and `config` services. We need to remove the proxy mixin from those at somepoint too because it's deprecated.